core/src/com/unciv/logic/civilization/managers/UnitManager.kt

```diff
@@ -71,9 +71,13 @@ class UnitManager(val civInfo:Civilization) {
         val unit = civInfo.gameInfo.tileMap.placeUnitNearTile(location, unitName, civInfo)
 
         if (unit != null) {
+            val triggerNotificationText = "due to gaining a [${unit.name}] "
```

Comment:
This is a new notification text so it needs to be added to template.properties (below "due to constructing [buildingName]")